### ISSUE TITLE
crash on upgrading flutter

### ISSUE THREAD

**Issue by DJafari** [vitusortner/floor#160]
after upgrade flutter to `1.7.10-pre.19` my app crashed on startup with this message :

```
E/AndroidRuntime( 5361): FATAL EXCEPTION: Sqflite
E/AndroidRuntime( 5361): Process: *, PID: 5361
E/AndroidRuntime( 5361): java.lang.RuntimeException: Methods marked with @UiThread must be executed on the main thread. Current thread: Sqflite
```

i thinks it's occurred because of this commit 

https://github.com/flutter/engine/commit/2c9e37c34e79475bbde7c8163eb5e56cdb9662a1

my setup for initialization :

```dart
void main() async {
    final db = await $FloorAppDatabase.databaseBuilder('app.db').build();
    runApp(AppComponent(db: db));
}
```

this is crash when run your example :

```
E/AndroidRuntime( 5859): FATAL EXCEPTION: Sqflite
E/AndroidRuntime( 5859): Process: io.vitus.example, PID: 5859
E/AndroidRuntime( 5859): java.lang.RuntimeException: Methods marked with @UiThread must be executed on the main thread. Current thread: Sqflite
E/AndroidRuntime( 5859): 	at io.flutter.embedding.engine.FlutterJNI.ensureRunningOnMainThread(FlutterJNI.java:794)
E/AndroidRuntime( 5859): 	at io.flutter.embedding.engine.FlutterJNI.invokePlatformMessageResponseCallback(FlutterJNI.java:727)
E/AndroidRuntime( 5859): 	at io.flutter.embedding.engine.dart.DartMessenger$Reply.reply(DartMessenger.java:140)
E/AndroidRuntime( 5859): 	at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler$1.success(MethodChannel.java:225)
E/AndroidRuntime( 5859): 	at com.tekartik.sqflite.SqflitePlugin$6.run(SqflitePlugin.java:778)
E/AndroidRuntime( 5859): 	at android.os.Handler.handleCallback(Handler.java:873)
E/AndroidRuntime( 5859): 	at android.os.Handler.dispatchMessage(Handler.java:99)
E/AndroidRuntime( 5859): 	at android.os.Looper.loop(Looper.java:193)
E/AndroidRuntime( 5859): 	at android.os.HandlerThread.run(HandlerThread.java:65)
I/Process ( 5859): Sending signal. PID: 5859 SIG: 9
```

**Comment by DJafari** [comment]
@vitusortner i opened a issue in sqflite repository ( https://github.com/tekartik/sqflite/issues/233 )

thanks to @alextekartik this bug fixed in `1.1.6+1` version, please update this dependency and publish new version

thanks

**Comment by MahdiPishguy** [comment]
@DJafari Thanks in advance to report this bug of library

**Comment by vitusortner** [comment]
Resolved by #169.
